software/contrib/master_clock.py

```diff
@@ -391,11 +382,12 @@ def loadState(self):
     async def main(self):
         while True:
             if not self.clockSelectionScreenActive:
-                # Display selected screen
-                if self.screen == 1:
-                    self.screen1()
-                else:
-                    self.screen2()
+                # # Display selected screen
+                # if self.screen == 1:
+                #     self.screen1()
+                # else:
+                #     self.screen2()
```

Comment:
I think it'd be better to remove this since the methods `screen1()` and `screen2()` have been removed.

Reply:
Agreed, good spot. Done.